django_cloud_deploy/cli/prompt.py

```diff
@@ -838,6 +860,205 @@ def _validate(self, s):
             raise ValueError('The provided billing account does not exist.')
 
 
+class GroupingPrompt(TemplatePrompt):
```

Comment:
What are the benefits of making this a class that gets inherited vs a utility function?

Reply:
Right now this utility function is only used in the two database prompts. Maybe it is not necessary to make it visible to other classes. But I am fine to make it a utility function.